src/molecules/formfields/ToggleField/index.js

```diff
@@ -57,17 +58,31 @@ ToggleField.propTypes = {
    * Label is optional. If not provided, component will reorganize accordingly.
    */
   label: PropTypes.string,
-  /**
-   * For use with the `button-toggle` type.
-   *
-   */
-  toggleChoices: PropTypes.array.isRequired,
-  onClick: PropTypes.func,
+  toggleChoices: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string,
+      value: PropTypes.oneOfType([
+        PropTypes.String,
+        PropTypes.Int,
+        PropTypes.Bool
+      ])
+    })
+  ),
   /**
    * The `tooltopMessage` can be anything that can be rendered:
    * `numbers`, `strings`, `elements` or an `array` (or fragment) containing these types.
    */
-  tooltipMessage: PropTypes.node
+  tooltipMessage: PropTypes.node,
+  /**
+   * @ignore
```

Comment:
hmm yeah the reason I ignored it was to prevent the defaultProp from showing the `onChange` function I wrote - I didn't want to require it, although maybe I should because it's the only way the component will work